habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -44,6 +45,24 @@ def __init__(self, config=None):
         self.envs = None
         if config is not None:
```

Comment:
This will need to be `self.config` or be done before the call to `super().__init__(config)`